concepts/ofd/index.md

```diff
@@ -0,0 +1,372 @@
+| Loss of current epoch block proposer rewards | n/a  | n/a | The _block proposer_ does not earn proposer rewards for a block outside the [_delta_ $\Delta$](/concepts/ofd/#delta-delta) if it includes an [_activity proof_](/concepts/ofd/#activity-proof) that does not contain signatures for $> \frac{2}{3}$ of block quorum voting power. The forfeited proposer rewards are simply not deducted from the total ATN and NTN rewards earned for the epoch and so remain in the ATN and NTN rewards distributed to the committee stake delegators at epoch end. See [Proposer reward calculation](/concepts/ofd/#proposer-reward-calculation) formula.|
+| Loss of future staking rewards - '_jailing_' | n/a | n/a | If the inactivity penalty applies [jailing](/glossary/#jailing) for the fault, then the validator is temporarily or permanently barred from selection to the consensus committee. The _offending validator_ loses the opportunity to earn future staking rewards as a committee member until it resumes an `active` state. Jail period determined by the [Jailing period calculation](/concepts/ofd/#jailing-period-calculation). |
+| Slashing of stake token | Autonity Protocol [`treasury`](/reference/genesis/#config.autonity-object) account | epoch end | The _offending validator's_ bonded stake is slashed for the omission penalty amount. Slashing is applied at epoch end according to the protocol's [Penalty-Absorbing Stake (PAS)](/glossary/#penalty-absorbing-stake-pas) model. Amount determined by the [Slashing amount calculation](/concepts/ofd/#slashing-amount-calculation). |
+
+
+### Rewards
+
+Economic rewards are provided to _block proposers_ and through the withholding of staking and Newton inflation rewards to the _community_.
+
+Proposer rewards are distributed as incentive for _block proposers_ that have included [activity proofs](/concepts/ofd/#activity-proof) in blocks where the number of signatures is greater than the $\frac{2}{3}$ quorum of voting power required for consensus. See [Proposer reward calculation](/concepts/ofd/#proposer-reward-calculation) for the formula.
+
+Withheld rewards distributed to the _community_ are conditional on the extent to which the _offending validator_ was inactive during the epoch, if it was jailed, and the amount of stake delegated to _offending validator_.
+
+::: {.callout-note title="Rewards are ATN and NTN" collapse="false"}
+
+Both _proposer rewards_ and _withheld rewards_ comprise ATN and NTN that the _offending validator_ has earned in the epoch:
+
+- ATN staking rewards from transaction fees, withheld for inactivity
+- NTN rewards from the Newton inflation mechanism, withheld for jailed validators.
+
+:::
```

Comment:
I am not sure what this callout tries to communicate, since it mixes up proposer rewards and withheld rewards

Reply:
Re-reading it, completely agree.

Deleted in commit f1aab07ed219300f67fcbadd84053bde435f6ebc